packages/react-native-win32/src/overrides.json

```diff
@@ -2,54 +2,38 @@
   "overrides": [
     {
       "type": "platform",
-      "file": "index.ts"
+      "file": "index.win32.ts"
     },
     {
       "type": "patch",
       "file": "Libraries\\Alert\\Alert.win32.js",
       "baseFile": "Libraries\\Alert\\Alert.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "dd77ba5b6da36da0aea70e638bf6bc2f88e4d073",
+      "baseVersion": "0.62.0-rc.3",
```

Comment:
>0.62.0-rc.3 [](start = 22, length = 11)

It's totally fine and expected that version numbers are all over the place. This is just the version where I first upgraded a file, and there hasn't been a change since.